src/app/cityio/cityio.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from "@angular/core";
+import { CityioService } from "./cityio.service";
+
+@Component({
+  selector: "app-cityio",
+  templateUrl: "./cityio.component.html",
+  styleUrls: ["./cityio.component.css"],
+  template: `
+    <h3>{{ title }}</h3>
+  `
+})
+export class CityioComponent implements OnInit {
+  constructor(private cityIOservice: CityioService) {}
+  title = "cityIO status...";
+  ngOnInit() {
+    this.getCityIOatInterval();
+  }
+
+  getCityIOatInterval() {
```

Comment:
why should the component handle the interval? or maybe, what is this component’s roll?